source/includes/fact-connection-strings.rst

```diff
@@ -1,11 +1,17 @@
-:ref:`mongosync <c2c-mongosync>` uses a standard :ref:`MongoDB URI
-connection string <mongodb-uri>` to connect clusters. 
+:ref:`mongosync <c2c-mongosync>` uses a :ref:`MongoDB URI
+connection string <mongodb-uri>` to connect clusters:
 
-The standard URI connection scheme has the form:
+- The SRV connection scheme has the form:
```

Comment:
[question] The JIRA ticket mentions that we could "also note that you can grab the SRV string from the Atlas UI: https://www.mongodb.com/docs/atlas/tutorial/connect-to-your-cluster-v2/" 

Should we note this somewhere—maybe under the code blocks?

Reply:
Fixed